### PR TITLE
feat(pages): Redesigned all default pages according to sketches discu…

### PR DESCRIPTION
**Pull request by Gawdfrey**
## ☕️ Reasoning

@ndom91 I managed to sync and close my original PR/fork so attempting again...

Based on [this PR](https://github.com/nextauthjs/next-auth/pull/5802) I went ahead and redesigned the pages according to the sketches we were discussing.

This PR introduces minimal code changes, but makes a few changes in the styling. Most notably is the introduction of a card background variable, the removal of box shadows and the usage of the button text variable (that had not been in use earlier?).

I have made every page (Signin, signout, verify-request and error) follow the same design.

## 🧢 Checklist

- [X] Documentation
- [X] Tests
- [X] Ready to be merged

## 🎫 Affected issues

Augments: https://github.com/nextauthjs/next-auth/pull/5802

## 📌 Resources

- [Security guidelines](../Security.md)
- [Contributing guidelines](../CONTRIBUTING.md)
- [Code of conduct](../CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

**Comment by Gawdfrey** [comment]
@ndom91 it seems as though the links to the provider logos have been broken.

Looking at the Apple provider, the url points to: "https://raw.githubusercontent.com/nextauthjs/next-auth/ndom91/login-btn-styling/packages/next-auth/provider-logos/apple.svg"

Where ".../ndom91/login-btn-styling/..." should have been replaced by ".../main/..."

**Comment by ndom91** [comment]
> @ndom91 it seems as though the links to the provider logos have been broken.
> 
> Looking at the Apple provider, the url points to: "https://raw.githubusercontent.com/nextauthjs/next-auth/ndom91/login-btn-styling/packages/next-auth/provider-logos/apple.svg"
> 
> Where **".../ndom91/login-btn-styling/..."** should have been replaced by **".../main/..."** 

Yeah good call, that should have been changed before merging. Thats my bad, fixing it now

**Comment by Gawdfrey** [comment]
Great!

Here are images of the different styled pages:

**Signin**

image 5
image 6

**Signout**

image 7
image 8

**Error**
image 9
image 10

**Verify request**
image 11
image 12

**Comment by ThangHuuVu** [comment]
I think it’s neat. For error & verifying request, IMO fixed width box looks better 🤔  WDYT @Gawdfrey

**Comment by Gawdfrey** [comment]
> I think it’s neat. For error & verifying request, IMO fixed width box looks better 🤔 WDYT @Gawdfrey

I do agree!

I went over and locked all cards to 350px on larger screens and noticed also that the sign in screen did not scale down to 320px which should be the minimum, so I fixed that as well.

File: packages/next-auth/src/css/index.css [hunk 1]
```diff
@@ -112,20 +114,22 @@ a.button {
...
   position: relative;
   display: flex;
   align-items: center;
   justify-content: center;
 
+  @media (max-width: 450px) {
+    font-size: 0.9rem;
+  }
+
   &:has(img) {
```
Comment (ThangHuuVu) [inline packages/next-auth/src/css/index.css:133]: `:has()` is not supported on Firefox, making the button image look tight. Can we make improvements for Firefox here?
Screen Shot 2022-11-30 at 22 04 11

Comment (Gawdfrey) [reply packages/next-auth/src/css/index.css]: Fixed now.
image

Targeting the span within the button works fine without the has conditional.

**Comment by Gawdfrey** [comment]
I am though having some issues with undefined adapters in the app.
image

Will have to test some more later what causes this.

**Review by ThangHuuVu — APPROVED** [review]
LGTM now! @ndom91 could you take another look before we merge? 💅

**Comment by vedantmgoyal2009** [comment]
@balazsorban44 can you approve release workflow so that I can take a look at this one?

**Comment by ndom91** [comment]
@Gawdfrey @ThangHuuVu just took another look at this. Works fine on my machine :tm: :partying_face: 

PR looks good to me too except for one tiny thing. @Gawdfrey could you also update the doc string you updated in the `versioned_docs/version-beta` version of that page? We plan on using that as main docs soon :pray: 

Other LGTM :100:

**Comment by Gawdfrey** [comment]
> @Gawdfrey @ThangHuuVu just took another look at this. Works fine on my machine ™️ 🥳
> 
> PR looks good to me too except for one tiny thing. @Gawdfrey could you also update the doc string you updated in the `versioned_docs/version-beta` version of that page? We plan on using that as main docs soon 🙏
> 
> Other LGTM 💯

Great!
I have changed the string now.

**Review by vedantmgoyal9 — APPROVED** [review]
@balazsorban44 @ndom91 Ready to be merged!

**Comment by ndom91** [comment]
Quick note to the others - no changes regarding docs necessary in this PR :partying_face:

File: packages/core/src/lib/pages/signin.tsx [hunk 2]
```diff
@@ -64,7 +71,17 @@ export default function SigninPage(props: {
           }}
         />
       )}
-      {theme.logo && <img src={theme.logo} alt="Logo" className="logo" />}
```
Comment (dcheepurupalli) [inline packages/core/src/lib/pages/signin.tsx:67]: This condition has been removed. Due to this new version, the logo is not showing after th econfiguration

**Pull request merged**